### ISSUE TITLE
Excel 365: Formula doesn't appear in Braille when editing a cell.

### ISSUE THREAD

**Issue by DGartmann** [nvaccess/nvda#11991]
### Steps to reproduce:
Open Excel to a blank workbook.
In cell A1 type the following formula: =5+4.
Press Tab to save it and move to the next cell.
Press Left Arrow to move back to A1.
Press F2 to go into edit mode to read the formula instead of just the result.

### Actual behavior:
In the first cell of the Braille display, the digit 9 is shown and on the following cells I get the information that there is a formula followed by the cell coordinates.

### Expected behavior:
The formula should be shown on the Braille display.

A work-around to read the formula on the Braille display:
When I press Alt with Tab to get out of Excel and Alt with Tab again to back into Excel, I see the formula on the Braille display.

#### NVDA installed/portable/running from source:
Installed.

#### NVDA version:
2020.4 beta 3.

#### Windows version:
Windows10.

#### Name and version of other software in use when reproducing the issue:
Microsoft Excel 365.

#### Other information about your system:

Focus40 Blue Braille display.
#### Does the issue still occur after restarting your computer?
Yes.

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
With NVDA 2020.3 the formula is not shown either. I have not tried the work-around described in this issue.

#### If addons are disabled, is your problem still occuring?

Yes.

#### Did you try to run the COM registry fixing tool in NVDA menu / tools?

Yes.

**Comment by msviolafangirl** [comment]
Hi, I just tested this. I found that once I entered =5+4 into a cell, the braille display did indeed display the digit 9, followed by indiciation that it's a formula, and the cell coordinates. Whenever I press F2 on a cell to edit it, I indeed see the formula on the braille display. I kinda don't think the full formulas are meant to be displayed unless you're editing the cell. Anyhow, I think this one just appears to be a very hard one to reproduce unfortunately.
